src/ai/backend/test/cli/__main__.py

```diff
@@ -116,7 +120,7 @@ def run_cli(
 @main.command()
 @click.pass_obj
 def get_all_specs(cli_ctx: CLIContext) -> None:
-    spec_manager = TestSpecManager.default()
+    spec_manager = TestSpecManager(specs=ROOT_TEST_SPECS)
```

Comment:
Let's make the TestSpecManager creation a private function, since it's always the same in the cli.